secure_message/resources/threads.py

```diff
@@ -113,16 +114,14 @@ def get():
 
 
 class ThreadCounter(Resource):
-    """Get count of unread messages using v2 endpoint"""
+    """Get count of all open or closed messages"""
     @staticmethod
     def get():
-        survey = request.args.getlist('survey')
-        if request.args.get('label'):
-            label = str(request.args.get('label'))
-            if label.lower() == 'unread':
-                return jsonify(name=label, total=Retriever.thread_count_by_survey(g.user, survey, label))
-            else:
-                logger.debug('Invalid label name', name=label, request=request.url)
-                raise BadRequest(description="Invalid label")
-        else:
-            return jsonify(total=Retriever.thread_count_by_survey(g.user, survey))
+        surveys = request.args.getlist('survey')
+        is_closed = bool(strtobool(request.args.get('is_closed')))
```

Comment:
We should default this to 'false' if not provided as otherwise a call to `/v2/messages/count` without the `is_closed` param fails.